jobfair-frontend/src/pages/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,37 @@
+import { ATTENDANT, COMPANY_EMPLOYEE, COMPANY_MANAGER } from '../../constants/RoleType';
+import { FAQComponent } from '../../components/customized-components/FAQ/FAQ.components';
+import { HomePageAdComponent } from '../../components/customized-components/HomePage/HomePageAd.component';
+import { useSelector } from 'react-redux';
+import HomePageHeaderComponent from '../../components/customized-components/HomePage/HomePageHeader.component';
+import React from 'react';
+
+const HomePage = () => {
+  const role = useSelector((state) => state.authentication.user.roles);
+
+  const handleDisplayHomePage = (role) => {
+    switch (role) {
+      case COMPANY_MANAGER:
```

Comment:
no need this switch case, there is already kick back default page handle in CAP-426